tensornetwork/tensor.py

```diff
@@ -111,7 +111,11 @@ def reshape(self, shape: Sequence[int]) -> "Tensor":
     """
     reshaped = self.backend.reshape(self.array, shape)
     return Tensor(reshaped, backend=self.backend)
-
+  def block(self):
```

Comment:
Is that standard terminology?

Reply:
Sorry but I couldn't understand your question can u please explain?

Reply:
The function name `block` is not very descriptive of what the function is actually doing. `item` would be more appropriate.

Reply:
Okay I would change it but how should I change it. I am really sorry that I am disturbing you by asking silly questions. Once I get to know the procedure I would change it. @mganahl Thank you for your feedback